README.yaml

```
@@ -67,7 +67,33 @@ description: |-
 # How to use this project
 usage: |-
   ```
-  TODO
+  module "atlantis" {
+    source = "git::https://github.com/cloudposse/terraform-aws-ecs-atlantis.git?ref=tags/0.1.0"
```

```suggestion
    source = "git::https://github.com/cloudposse/terraform-aws-ecs-atlantis.git?ref=master"
```
We usually use master branch in examples